src/System.CommandLine.Tests/ParseDirectiveTests.cs

```diff
@@ -82,5 +82,25 @@ public async Task When_there_are_errors_then_parse_directive_sets_exit_code_1()
 
             exitCode.Should().Be(1);
         }
+
+        [Fact]
+        public async Task When_there_are_errors_then_parse_directive_sets_exit_code_to_custom_value()
+        {
+            var command = new RootCommand
+            {
+                new Option("-x")
```

Comment:
Please use `Option<int>` so there's less code to refactor assuming we remove the `Argument` property.

Reply:
Good point. Fixed up all of the tests in this file.